.gitignore

```diff
@@ -9,3 +9,4 @@
 TODO
 test/app.json
 /docs
+tutorials/ExternallApplication.wrap.md
```

Comment:
why are we ignoring this specific file?

Reply:
I I did not get around to getting to do something with that  tutorial at the moment

Reply:
can we just remove it instead of adding ignore rules specifically for it.